acid-state-exts/src/Serokell/AcidState/Instances.hs

```diff
@@ -25,9 +23,6 @@ instance (Eq a, Hashable a, SafeCopy a) => SafeCopy (HashSet a) where
     putCopy = contain . safePut . HS.toList
     getCopy = contain $ HS.fromList <$> safeGet
 
-instance (Eq a, Hashable a, SafeCopy a, SafeCopy b) => SafeCopy (HashMap a b) where
-    putCopy = contain . safePut . HM.toList
-    getCopy = contain $ HM.fromList <$> safeGet
```

Comment:
I'm actually not sure that  moving this "downstream" into `core` is the right idea. For things that are defined outside of our "universe", keeping them as upstream as possible makes sense to me. What do you think?

Reply:
That makes sense, thanks for clarifying :)